src/com/goide/dlv/protocol/DlvApi.java

```diff
@@ -94,6 +98,10 @@
     public Function function;
   }
 
+  public static class StacktraceOut {
```

Comment:
These wrappers are now present in delve as well. If there's a way to access the information directly without the wrapper that would be great (but unfortunately I couldn't see how to do this).